sdcm/nemesis.py

```diff
@@ -1653,27 +1653,26 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
+        if not (self.cluster.params.get('use_mgmt') or
+                self.cluster.params.get('use_cloud_manager')):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
         cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        backup_bucket_location = self.cluster.get_backup_buckets_name()
         if cluster_backend == 'aws':
-            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'gce':
-            bucket_name = f"gcs:{backup_bucket_location}"
+            location_list = [f"s3:{backup_bucket_location.split()[0]}"]
+        elif cluster_backend == 'gce':  # For cloud it should be "elif 'gce' in cluster_backend:
+            location_list = [f"gcs:{backup_bucket_location}"]
+        elif cluster_backend == 'aws-siren':
+            location_list = None
```

Comment:
why this is set to None ? wasn't it the whole point of introducing `get_backup_buckets_name()` to fill it in this case ?

Reply:
it turns out that on cloud you don't need to specify the bucket, the get_backup_buckets_name will be removed